### PR TITLE
phpstan 0.11

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

File: src/Schema/Validation/IdValidation.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ protected function validateDuplicities(SchemaBuilder $builder): void
 			foreach ($controller->getMethods() as $method) {
 
 				// Skip if @Id is not set
-				if (empty($method->getId())) continue;
```
Comment (f3l1x) [inline src/Schema/Validation/IdValidation.php]: Why?

Comment (mabar) [reply src/Schema/Validation/IdValidation.php]: Trying to throw away all `empty()`, it's too magical.
E.g. `empty('0') === true`. And `'0'` is valid id.

Comment (f3l1x) [reply src/Schema/Validation/IdValidation.php]: I see

**Comment by f3l1x** [comment]
It's failing. Please fix it.

**Comment by mabar** [comment]
@f3l1x One of the builds runned with old, actually overriden commit. Not sure how it's possible, but code is ok. It should pass with merge build.

**Comment by f3l1x** [comment]
I see. Merged.

**Pull request merged**